src/sentry/static/sentry/app/index.js

```diff
@@ -1,153 +1,15 @@
-import Reflux from 'reflux';
-import * as Router from 'react-router';
-import * as Sentry from '@sentry/browser';
-import {ExtraErrorData, Tracing} from '@sentry/integrations';
-import createReactClass from 'create-react-class';
-import jQuery from 'jquery';
-import moment from 'moment';
-
-import {metric} from 'app/utils/analytics';
-import ConfigStore from 'app/stores/configStore';
-import Main from 'app/main';
-import ajaxCsrfSetup from 'app/utils/ajaxCsrfSetup';
-import plugins from 'app/plugins';
-
-// SDK INIT  --------------------------------------------------------
-// window.__SENTRY__OPTIONS will be emmited by sdk-config.html before loading this script
-Sentry.init({
-  ...window.__SENTRY__OPTIONS,
-  integrations: [
-    new ExtraErrorData({
-      // 6 is arbitrary, seems like a nice number
-      depth: 6,
-    }),
-    new Tracing({
-      tracingOrigins: ['localhost', 'sentry.io', /^\//],
-      autoStartOnDomReady: false,
-    }),
-  ],
-});
-
-Sentry.configureScope(scope => {
-  if (window.__SENTRY__USER) {
-    scope.setUser(window.__SENTRY__USER);
-  }
-  if (window.__SENTRY__VERSION) {
-    scope.setTag('sentry_version', window.__SENTRY__VERSION);
-  }
-});
-
-// -----------------------------------------------------------------
-
-// Used for operational metrics to determine that the application js
-// bundle was loaded by browser.
-metric.mark('sentry-app-init');
-
-// setup jquery for CSRF tokens
-jQuery.ajaxSetup({
-  //jQuery won't allow using the ajaxCsrfSetup function directly
-  beforeSend: ajaxCsrfSetup,
-});
-
-// App setup
+// Initial data hydration. The __initialData will be set by the django backend
+// serving sentry.
 if (window.__initialData) {
-  ConfigStore.loadInitialData(window.__initialData, window.__languageCode);
-}
-
-// these get exported to a global variable, which is important as its the only
-// way we can call into scoped objects
+  const {distPrefix, csrfCookieName, sentryConfig, userIdentity} = window.__initialData;
 
-const render = Component => {
-  const rootEl = document.getElementById('blk_router');
-
-  try {
-    ReactDOM.render(<Component />, rootEl);
-  } catch (err) {
-    // eslint-disable-next-line no-console
-    console.error(
-      new Error(
-        'An unencoded "%" has appeared, it is super effective! (See https://github.com/ReactTraining/history/issues/505)'
-      )
-    );
-    if (err.message === 'URI malformed') {
-      window.location.assign(window.location.pathname);
-    }
-  }
-};
-
-// The password strength component is very heavyweight as it includes the
-// zxcvbn, a relatively byte-heavy password strength estimation library. Load
-// it on demand.
-async function loadPasswordStrength(callback) {
-  const module = await import(/* webpackChunkName: "passwordStrength" */ 'app/components/passwordStrength');
-  callback(module);
+  window.csrfCookieName = csrfCookieName;
+  window.__sentryGlobalStaticPrefix = distPrefix;
+  window.__SENTRY__OPTIONS = sentryConfig;
```

Comment:
Do we still need these globals?

Reply:
Maybe only the cookie one here: https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/constants/index.jsx#L102

Reply:
But I would rather refactor them in another PR I think